src/blockchain/milestones.js

```diff
@@ -168,7 +168,7 @@ const milestonesFactory = app => {
       if (data.length === 1) {
         const m = data[0];
         const { from } = await getTransaction(app, txHash);
-        await milestones.patch(
+        return milestones.patch(
```

Comment:
This change and others like this stops us logging potential error on catch statement. I am not sure however, but it's probable.
You can create a variable to hold patch return value (`const milestone = `) and return it.

Reply:
> This change and others like this stops us logging potential error on catch statement. I am not sure however, but it's probable.
> You can create a variable to hold patch return value (`const milestone = `) and return it.

You are right that's a better idea, 
I change in this way